docs/source/pages/tutorials/npm_provenance.rst

```diff
@@ -0,0 +1,127 @@
+.. Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved.
+.. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+--------------------------------------------------
+Provenance discovery, extraction, and verification
+--------------------------------------------------
+
+This tutorial demonstrates how Macaron can automatically retrieve provenance for NodeJS artifacts, validate the contents, and verify the authenticity. Any artifact that can be analyzed and checked for these properties can then be trusted to a greater degree than would be otherwise possible, as provenance files provide an assurance that the artifact is what it claims to be, and when verified, this becomes a certainty.
+
+For NodeJS artifacts, Macaron makes use of available features provided by `npm <https://npmjs.com/>`_. Most importantly, npm allows developers to generate provenance files when publishing their artifacts. The `semver <https://www.npmjs.com/package/semver>`_ package is chosen as an example for this tutorial.
+
+******************************
+Installation and Prerequisites
+******************************
+
+Skip this section if you already know how to install Macaron.
+
+.. toggle::
+
+    Please follow the instructions :ref:`here <installation-guide>`. In summary, you need:
+
+        * Docker
+        * the ``run_macaron.sh``  script to run the Macaron image.
+
+    .. note:: At the moment, Docker alternatives (e.g. podman) are not supported.
+
+
+    You also need to provide Macaron with a GitHub token through the ``GITHUB_TOKEN``  environment variable.
+
+    To obtain a GitHub Token:
+
+    * Go to ``GitHub settings`` → ``Developer Settings`` (at the bottom of the left side pane) → ``Personal Access Tokens`` → ``Fine-grained personal access tokens`` → ``Generate new token``. Give your token a name and an expiry period.
+    * Under ``"Repository access"``, choosing ``"Public Repositories (read-only)"`` should be good enough in most cases.
+
+    Now you should be good to run Macaron. For more details, see the documentation :ref:`here <prepare-github-token>`.
+
+********
+Analysis
+********
+
+To perform an analysis on the latest version of semver (when this tutorial was written), Macaron can be run with the following command:
+
+.. code-block:: shell
+
+    ./run_macaron.sh analyze -purl pkg:npm/semver@7.6.2 --skip-deps
```

Comment:
How about using  `pkg:npm/semver@7.6.0` [used in the integration tests](https://github.com/oracle/macaron/tree/19e6924fe28ddf0a9cfada1750881b7978a93023/tests/integration/cases/semver)? Alternatively, we can change the version in the integration test to save you from updating some of the screenshots :slightly_smiling_face:

Reply:
Good idea. I'll update the integration tests in that case!